pkg/assetfetcher/asset_test.go

```diff
@@ -47,6 +47,30 @@ func TestLastAssessedForVulnerabilities(t *testing.T) {
 			time.Time{},
 			true,
 		},
+		{
+			"empty timestamp field",
+			assetHistoryEvents{
+				AssetHistory{Type: "SCAN", Date: ""},
+			},
+			time.Time{},
+			true,
+		},
+		{
+			"0 timestamp field",
+			assetHistoryEvents{
+				AssetHistory{Type: "SCAN", Date: "0001-01-01T00:00:00Z"},
+			},
+			time.Time{},
+			false,
```

Comment:
Question: shouldn't we expect an error with a zero value timestamp?

Reply:
Good question, lastScannedTimestamp() won't return an error with this case, it will return the zero value timestamp. However, the only method that calls lastScannedTimestamp() is AssetPayloadToAssetEvent(), and that checks if lastScannedTimestamp() returns a zero value, which will then result in an error.